### ISSUE TITLE
possible bug with formulas in SES satisfaction

### ISSUE THREAD

**Issue by moocunsw** [moocunsw/PVCE-QILT#29]
@whyes; when you have a chance, could you please go through the way in which the overall satisfaction for SES is calculated?

It seems that we have some internal inconsistencies between the tab labeled ‘Compare SES 2015-16’ and the ‘Compare SES with others’ (use GO8 for reference which seems correct)
 
In the latter the figure is 76% which matches with the external QILT site, the former has slightly higher values around 78% which is not right. I suspect that the dataframe behind them is different and in the year comparison the N responses do not match.

**Comment by whyes** [comment]
@moocunsw yes the two different sections actually get their data from 2 different tables. It seems to display data for two different things:

1. The undergraduate SES data for 2015 and 2016
2. The average SES data over 2015/2016 and undergraduate/postgraduate

Compare SES 2015-2016 uses:
`SES_undergraduate_2015_index_university` and `SES_undergraduate_2016_index_university`
Overall satisfaction for UNSW is 78% for both 2015 and 2016 from "QEE_AVG" column

Compare SES with others (G08 tab) uses:
`Compare SES_national_2015_2016_QILT`
Overall satisfaction for UNSW is 76% from "satoq" column

These tables have different values for the different indicators. I'm not sure which one should be the correct one used. There is currently no tables in the database for postgraduate SES data.
